app/models/rules_of_origin/link.rb

```diff
@@ -1,10 +1,13 @@
 # frozen_string_literal: true
 
+require 'digest'
+
 module RulesOfOrigin
   class Link
     include ActiveModel::Model
 
     attr_accessor :text, :url
+    attr_writer :id
```

Comment:
Do you need this?

Reply:
It does get used as part of the specs I think, and seems generally useful